index.js

```diff
@@ -43,7 +43,14 @@ module.exports = {
           },
         ],
         '@typescript-eslint/explicit-module-boundary-types': 'off',
-        '@typescript-eslint/no-unused-vars': 'warn',
+        '@typescript-eslint/no-unused-vars': [
+          'warn',
+          {
+            varsIgnorePattern: '^_',
+            argsIgnorePattern: '^_',
+            ignoreRestSiblings: true,
+          },
+        ],
```

Comment:
when add `_` character at the beginning of the variable name, eslint should ignore this rule.